ui/src/main/java/edu/wpi/grip/ui/OperationListController.java

```diff
@@ -34,15 +34,14 @@
   @FXML private Tab root;
   @FXML private VBox operations;
   @Inject private OperationController.Factory operationControllerFactory;
-  private String baseText = null;
   private ControllerMap<OperationController, Node> operationsMapManager;
 
   @FXML
   protected void initialize() {
     operationsMapManager = new ControllerMap<>(operations.getChildren());
-    baseText = root.getText();
 
     InvalidationListener filterOperations = observable -> {
+      String baseText = root.getText();
```

Comment:
This will actually break the code. Run GRIP and try using the search functionality. You'll understand what I mean.